tests/integration/test_s3.py

```diff
@@ -739,7 +744,8 @@ def test_verify_can_switch_sigv4(self):
         service = self.session.get_service('s3')
         endpoint = service.get_endpoint('us-east-1')
         # The set_config_variable should ensure that we use sigv4 for s3.
-        self.assertIsInstance(endpoint.auth, botocore.auth.S3SigV4Auth)
+        # TODO: How do we check this? It's not exposed anywhere anymore...
+        #self.assertIsInstance(endpoint.auth, botocore.auth.S3SigV4Auth)
```

Comment:
Wait, this is a integration test. Scratch my suggestion to mock stuff out. Just make a call using KMS. Sigv4 is required for that and you will get errors if you do not use it.